nupic/research/frameworks/dendrites/routing/utils.py

```diff
@@ -34,6 +33,10 @@
 ]
 
 
+def generate_context_integers(num_contexts):
```

Comment:
I understand you are building another PR with the code optimizations - if this is related to the optimizations, maybe we should move it there, so this PR is specific to profiling